CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# Version 0.1.0
```

Comment:
I don't think there's any reason to start incrementing the release number yet, we haven't released anything yet :D